web/src/connectors/documents-sidebar-connector/documents-sidebar-connector.tsx

```diff
@@ -41,7 +41,7 @@ export const DocumentsSidebarConnector = <T extends object>({
     filters
 }: DocumentsSidebarConnectorProps<T>) => {
     const [searchText, setSearchText] = useState('');
-    const [sortConfig, setSortConfig] = useState({
+    const [sortConfig] = useState({
```

Comment:
If we are going to use it this way, you'll actually create immutable `sortConfig`, since there will be no way to update it. 
Since `sortConfig` relies on `sortField` probably it's better to store it in constant e.g `const sortConfig = {...}` in that case if component will be re-rendered and `sortField` will be updated, the sortConfig will be updated in accordance with the changes